src/extension/ddappsec.c

```diff
@@ -273,11 +283,22 @@ static int _do_rinit(INIT_FUNC_ARGS)
         return SUCCESS;
     }
 
-    // request_init
-    int res = dd_request_init(conn);
+    int res = 0;
```

Comment:
```suggestion
    int res = dd_success;
```
Not a big deal but you might as well make it obvious.

Reply:
Done